src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -127,7 +131,6 @@
         }
 
         .datagrid-column {
-            position: relative;
```

Comment:
We have had the 
```
container (with position: relative) + child (with position:absolute)
``` 
as a pattern for popovers. However, this results in popovers clipping. This is removed as we are now using our popover implementation for the filter popover rather than relying on the `position: relative`.